### PR TITLE
Feat add esign document upload

### PR DESCRIPTION
**Pull request by indraprasetya154**
## Overview
Feat add esign document upload
- List document type
- List signer/receiver
- Check status by tid's
- Create esign document upload

## Evidence:
project: SIDEBAR
title: Feat add esign document upload for elogistik
participants: @indraprasetya154 @azophy @fajarhikmal214

### CODE REVIEW

File: src/app/Helpers/Helpers.php [hunk 1]
```diff
@@ -24,6 +27,32 @@ function parseSetLocaleDate($value, $locale, $format)
     return Carbon::parse($value)->locale($locale)->translatedFormat($format);
 }
 
+function parseJWTToken($token)
```
Comment (azophy) [inline src/app/Helpers/Helpers.php:30]: this is not used anymore right @fajarhikmal214 ?

File: src/.env.example [hunk 2]
```diff
@@ -95,6 +95,12 @@ MYSQL_USER_LOG_ACTIVITY=false
 DOCKER_APP=
 MAXIMUM_MULTIPLE_ESIGN=10
 
+KEYCLOAK_BASE_URL=
+KEYCLOAK_REALM=
+KEYCLOAK_ISS=
+KEYCLOAK_CERTIFICATE_TTL=
+
+JWT_LEEWAY=60
```
Comment (azophy) [inline src/.env.example:103]: and this

File: src/app/Http/Middleware/EnsureTokenIsValid.php
Comment (azophy) [inline src/app/Http/Middleware/EnsureTokenIsValid.php:1]: this is also

File: src/composer.json
Comment (azophy) [inline src/composer.json:1]: this is also

File: src/config/jwt.php
Comment (azophy) [inline src/config/jwt.php:1]: so is this

File: src/app/Helpers/Helpers.php (hunk 1 above)
Comment (fajarhikmal214) [reply src/app/Helpers/Helpers.php:30]: Yes, I try to find this code in `development` branch and was not found. I think should be deleted

**Review by azophy — APPROVED** [review]

**Review by fajarhikmal214 — APPROVED** [review]

**Pull request merged**